assets/xml/objects/object_zl4.xml

```diff
@@ -0,0 +1,51 @@
+<Root>
+    <File Name="object_zl4" Segment="6">
+        <Skeleton Name="gChildZeldaCYSkel_00E038" Type="Flex" LimbType="Standard" Offset="0xE038"/>
+        <Animation Name="gChildZeldaCYAnim_00F0A4" Offset="0xF0A4"/>
```

Comment:
Courtyard. This is opposed to Zl1 that's used in cutscenes. Honestly, this object stuff is still a little WIP.

Reply:
Just using Courtyard should be fine, most people won't know what "CY" means so the abbreviation is not worth it because it only makes it harder to understand.

Reply:
We've since switched to just ChildZelda, as it turns out Zl1 is effectively unused and thus there's no ambiguity.